scripts/MultiPlotting/gridspec_engine.py

```diff
@@ -29,9 +29,10 @@ def __init__(self, grid=defaultGridspecGrid, max_plot=None):
     def getGridSpec(self, number):
         if number <= 0:
             return
-        if number > self._max_plot and self._max_plot is not None:
```

Comment:
I think just switching the order of these two checks should have the same effect as nesting them.